mmv1/templates/terraform/examples/clouddeploy_delivery_pipeline_basic.tf.erb

```diff
@@ -0,0 +1,10 @@
+resource "google_clouddeploy_delivery_pipeline" "<%= ctx[:primary_resource_id] %>" {
+  name = "<%= ctx[:vars]['delivery_pipeline'] %>"
+  location = "us-central1"
+  serial_pipeline  {
+    stages {
+      target_id = "test"
+      profiles = []
```

Comment:
where did this `profile` fields come from? There does not seem to be a corresponding field in the schema.

Reply:
oops. I should add [correct stages](https://cloud.google.com/deploy/docs/api/reference/rest/v1/projects.locations.deliveryPipelines#DeliveryPipeline.SerialPipeline) into `DeliveryPipeline.yaml`

Reply:
this example should not be needed at all. You can delete the file and `examples` block in the YAML